data/human/human missions.txt

```diff
@@ -4175,3 +4175,166 @@ mission "Hai Wormhole Warning"
+				`	"Have you had any Greek statues pass by through here?"`
+			`	The clerk laughs. "You're trying to get Alexandria's shipment back, aren't you?" Before you can answer, he continues. "If you want my client Lokust's statues, I'm afraid you're gonna have to pay a pretty penny for them. 500,000 credits is his asking price."`
+			choice
+				`	"Alright, I'll pay."`
+					goto pay
+				`	"Can't we negotiate?"`
+				`	"I can't pay that much!"`
+			`	"Sorry, his price is firm. I'm just the dealer. He's the one pulling the strings." A smirk begins to crawl across his face, and he leans in. "Look, I have no love for Lokust. Always jacking up his cut of the sale, threatening my customers, you get the idea. Not to mention my debts to him." The last part he says in a less bold tone. "What I'm saying is, is that I wouldn't mind if someone like you took him out. He's a real wacko, always going on about duels and other bull. If I were you, I'd challenge him to one. If he doesn't accept, call him a coward. That's sure to get him riled up." He wheezes in a way vaugely reminiscent of a laugh. "As a reward, I'll give you the statues for free if you come back, but you gotta take out his whole crew, right? Don't want any of his underlings coming for me."`
+			choice
+				`	"Never mind, I'll pay."`
+					goto pay
+				`	"All right, I'll fight him. `
+			`	"Heh, good choice. If you're worried about his crew, don't. I'm sure if Alexandria thinks you're good enough to hire, you can handle it. I'll call him up so you don't have to go chasing him down." He pulls up a hologram communicator and noticeably stiffens up. "Hello Lokust. Hope you're not too busy right now, because someone wants to see you." He turns the communicator towards you, and you see a man in clothes more befitting of a prince than a pirate.`
+			choice
+				`	"Fight me."`
+				`	"I've heard you're willing to duel over those statues you stole from the Library of Alexandria?"`
+			`	The communicator's audio quality is evidently not the highest, as his laugh is barely more than static. "I don't just throw hands with anyone, you see. That would make me little better than some barroom drunk. I have standards. Why should I waste my time with you?"`
+			choice
+				`	"Oh, I see. You're too cowardly to fight."`
+				`	"I didn't realize the great Lokust was such a weakling."`
+			`	He becomes visibily enraged the moment you're finished talking. "I will not stand these insults to my honor! We meet in the sky. Bring all of your weaponry to bear, for it is the last time you will get to use them."`
+			`	The communication cuts off. "He always uses that same line," the clerk says, rolling his eyes.`
+			apply
+				set "Duel Lokust"
+			decline
+			label pay
+			action
+				payment -500000
+			`	He looks almost disappointed. "Alright, here's the statues. Don't forget to tell any friends that might be redecorating about our low prices."`
+				decline
+
+mission "Saving Artifacts 3"  # Duel branch
```

Comment:
Just to confirm - putting the `#` on the same line doesn't mess up this mission offering, right? Just want to enure it's been tested.

Reply:
I haven't tested this mission, but I'm fairly certain this will not affect the mission. I didn't actually write this bit, pilover's responsible for these lines. The `#` is an artifact of that, and was put in so that I could understand what the code actually did. I can take it out if that would be easier.

Reply:
Sure, let's take them out just to be safe. Thanks!

Reply:
```suggestion
mission "Saving Artifacts 3"
```